core/src/main/java/com/google/common/truth/AbstractVerb.java

```diff
@@ -99,29 +117,40 @@ public S that(T target) {
     }
   }
 
-  protected static class MessagePrependingFailureStrategy extends FailureStrategy {
+  /**
+   * A {@link FailureStrategy} which delegates to an original strategy, and overrides the message
+   * when the various {@code fail()} methods are called, prepending the supplied formatted string
+   * to the Subject-supplied error message.
+   */
+  public static class DelegatingFailureStrategy extends FailureStrategy {
```

Comment:
Sorry - I was doing an instanceof check elsewhere, but mvoed it in here.  You're right.  It should be private.  Thanks!